cv_next/app/actions/comments/commentLikes.ts

```diff
@@ -0,0 +1,42 @@
+"use server";
+
+import { updateComment } from "@/server/api/comments";
+import CommentModel, { ClientCommentModel } from "@/types/models/comment";
+
+export const addLikeToComment = async (comment: ClientCommentModel, userId: string)
```

Comment:
maybe we should refactor this file cause these two functions look almost the same